provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -299,6 +299,19 @@ spec:
     served: true
     storage: true
 ---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Please rework the if checks above to avoid repeating the CRD definition

Reply:
removed the check for podif to resuse the crd in non-overlay case as well.